packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -29,14 +29,17 @@ import { customNamespaces } from './customNamespaces';
  */
 
 const ADDITIONAL_ALPHA_CHARS = ['@'];
+const CUSTOM_CONFIG = {
+  matrix: 'Array', // Have mathjs treat array-like objects as plain arrays rather than mathjs Matrix
+};
```

Comment:
I've been looking for this option for **AGES!** Basically by default `mathjs` will treat arrays as `Matrix` type, eg.
` typeof ([1, 2]) => Matrix`

This makes things really painful when combining with our own custom functions since we constantly have to cast matrixes to arrays. Now it will just treat matrixes as arrays automatically!